### PR TITLE
Remove unused named return variables

### PR DESCRIPTION
**Pull request by Amxx**
Fixes LIB-1033

For consistency with the rest of the codebase, removing unused named variables from the methods' signatures

### CODE REVIEW

**Review by frangio — APPROVED** [review]

File: contracts/access/extensions/AccessControlDefaultAdminRules.sol [hunk 1]
```diff
@@ -189,7 +189,7 @@ abstract contract AccessControlDefaultAdminRules is IAccessControlDefaultAdminRu
     /**
      * @inheritdoc IAccessControlDefaultAdminRules
      */
-    function pendingDefaultAdminDelay() public view virtual returns (uint48 newDelay, uint48 schedule) {
+    function pendingDefaultAdminDelay() public view virtual returns (uint48, uint48) {
         schedule = _pendingDelaySchedule;
```
Comment (frangio) [inline contracts/access/extensions/AccessControlDefaultAdminRules.sol:193]:
```suggestion
        uint48 schedule = _pendingDelaySchedule;
```

**Review by frangio — CHANGES_REQUESTED** [review]

**Comment by frangio** [comment]
We are probably using the named return values in tests.

**Comment by Amxx** [comment]
> We are probably using the named return values in tests.

Yes, we do ```const { newDelay, schedule } = await this.accessControl.pendingDefaultAdminDelay();```

**Review by frangio — APPROVED** [review]
I had added a new guideline:
> Functions with multiple return values should use named return values for clarity, even if the function uses an explicit `return` statement.

But reverted it because I don't think we should add the guideline without reviewing the rest of the codebase.

This is our intention though.

**Pull request merged**